packages/tasks/src/tasks/image-text-to-text/data.ts

```diff
@@ -63,12 +63,12 @@ const taskData: TaskDataCustom = {
 			id: "Qwen/Qwen2.5-VL-7B-Instruct",
 		},
 		{
-			description: "Image-text-to-text model with reasoning capabilities.",
-			id: "Qwen/QVQ-72B-Preview",
+			description: "Image-text-to-text model with agentic capabilities.",
+			id: "microsoft/Magma-8B",
```

Comment:
```suggestion
			id: "ibm-granite/granite-vision-3.2-2b",
```

Too much MSFT, showing variety in orgs is nice IMO

Reply:
we don't have a lot of reasoning/agentic models :/ I'd rather keep it

Reply:
I added granite separately